module-sys/SystemManager/include/SystemManager/CpuStatistics.hpp

```diff
@@ -1,26 +1,47 @@
-// Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 // For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 
 #pragma once
 
+#include "SystemManager/PowerManager.hpp"
+#include "SystemManager/CpuPrinter.hpp"
 #include <cstdint>
 
+extern "C" {
+#include "prof.h"
+}
+
+
+#define RUNTIME_STATS_FROM_RTOS
```

Comment:
Move to cmake as option

Reply:
yep, this is related to: https://github.com/mudita/MuditaOS/pull/3594#discussion_r808810506 curently I do not know if we want to track it all the time, or not :)